### PR TITLE
APIGOV-19207 Update SDK version

### PR DESCRIPTION
**Pull request by sagarkal**

### CODE REVIEW

**Comment by dfeldick** [comment]
1) updated SDK to last released version (1.1.3). Also updated some other dependencies
2) Fixed unit test mocks and some tests themselves so tests would run
3) Fixed many lint warnings

File: pkg/traceability/eventprocessor_test.go [hunk 1]
```diff
@@ -33,10 +34,10 @@ const (
...
 )
 
-func init() {
-	setupRedaction()
-	setupConfig()
-}
+// func init() {
+// 	setupRedaction()
+// 	setupConfig()
+// }
```
Comment (jcollins-axway) [inline pkg/traceability/eventprocessor_test.go:40]: Dead code?

**Review by sbolosan — APPROVED** [review]

**Review by jcollins-axway — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -43,4 +43,4 @@ build-trace-docker:
 
 test:
 	mkdir -p coverage
-	@go test -race -short -count=1 -coverprofile=coverage/coverage.cov ${GO_PKG_LIST}
```
Comment (tsjohns9) [inline Makefile:46]: Why was this removed?

Comment (dfeldick) [reply Makefile:46]: because according to the docs, -race is not even a valid flag for go test. It is only a go build flag. Unless you know differently...

Comment (dfeldick) [reply Makefile:46]: well, now I found another doc that says it is available for build, clean, get, install, list, run, and test commands. I'll put it back.

Comment (dfeldick) [reply Makefile:46]: but I guess that begs the question as to why it is only on make test, and not make build also.

Comment (tsjohns9) [reply Makefile:46]: Build flags are valid for running tests. I think we should keep it. I have found value in the -race flag when writing tests with channels and go routines.
